### PR TITLE
converting HTML to md

### PR DESCRIPTION
**Pull request by keharper**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [ ] Content fix or rewrite
- [x] Bug fix or improvement

## Summary

When this pull request is merged, the Getting Started guide will not contain HTML.

### CODE REVIEW

File: guides/v2.1/get-started/authentication/gs-authentication-oauth.md [hunk 1]
```diff
@@ -75,35 +71,16 @@ A request token is a temporary token that the user exchanges for an access token
...
+
+
+Parameter | Description
+--- | ---
+`oauth_consumer_key` | The consumer key is generated when you create the integration.
+`oauth_signature_method` | The name of the signature method used to sign the request. Must be the value `HMAC-SHA1`.
+`oauth_signature` | A generated value (signature)
+`oauth_nonce` | A random value that is uniquely generated by the application.
+`oauth_timestamp` | A positive integer, expressed in the number of seconds since January 1, 1970 00:00:00 GMT.
+`oauth_version` | The OAuth version.
```
Comment (jeff-matthews) [inline guides/v2.1/get-started/authentication/gs-authentication-oauth.md]: Add the following inline attribute to tables per style guide

> {:style="table-layout:auto;"}

File: guides/v2.1/get-started/authentication/gs-authentication-oauth.md [hunk 2]
```diff
@@ -121,43 +98,17 @@ The request token must be exchanged for an access token. Use the following API t
...
+Parameter | Description
+--- | ---
+`oauth_consumer_key` | The consumer key value that you retrieve after you register the integration.
+`oauth_nonce` | A random value that is uniquely generated by the application.
+`oauth_signature` | A generated value (signature)
+`oauth_signature_method` | The name of the signature method used to sign the request. Must be the value `HMAC-SHA1`.
+`oauth_timestamp` | A positive integer, expressed in the number of seconds since January 1, 1970 00:00:00 GMT.
+`oauth_version` | The OAuth version.
+`oauth_token` | The `oauth_token` value, or request token, obtained in [Get a request token](#pre-auth-token).
+`oauth_verifier` | The verification code that is tied to the consumer and request token. It is sent as part of the initial POST operation when the integration is activated.
```
Comment (jeff-matthews) [inline guides/v2.1/get-started/authentication/gs-authentication-oauth.md]: Add the following inline attribute to tables per style guide

> {:style="table-layout:auto;"}

File: guides/v2.1/get-started/authentication/gs-authentication-token.md [hunk 3]
```diff
@@ -58,36 +54,11 @@ A cron job that runs hourly removes all expired tokens.
...
-      </td>
-   </tr>
-</table>
+Component | Specifies
+--- | ---
+Endpoint |  A combination of the _server_ that fulfills the request, the web service, and the `resource` against which the request is being made.<br/><br/>For example, in the `POST https://magento.host/index.php/rest/V1/integration/customer/token` endpoint:<br/>The server is `magento.host/index.php/`,<br/> the web service is `rest`.<br/> and the resource is `/V1/integration/customer/token`.
+Content type | The content type of the request body. Set this value to either `"Content-Type:application/json"` or `"Content-Type:application/xml"`.
+Credentials | The username and password for a Magento account.<br/><br/>To specify these credentials in a JSON request body, include code similar to the following in the call: <br/><br/>`{"username":"<USER-NAME>;", "password":"<PASSWORD>"}`<br/><br/>To specify these credentials in XML, include code similar to the following in the call:<br/><br/>`<login><username>customer1@example.com</username><password>customer1pw</password></login>`
```
Comment (jeff-matthews) [inline guides/v2.1/get-started/authentication/gs-authentication-token.md]: Add the following inline attribute to tables per style guide

> {:style="table-layout:auto;"}

File: guides/v2.1/get-started/authentication/oauth-errors.md [hunk 4]
```diff
@@ -1,103 +1,29 @@
...
+400 | 4 | `timestamp_refused` | The timestamp value in the oauth_timestamp parameter is incorrect.
+401 | 5 | `nonce_used` | The nonce-timestamp combination has already been used.
+400 | 6 | `signature_method_rejected`| The signature method is not supported. The following methods are supported: HMAC-SHA1.
+401 | 7 | `signature_invalid` | The signature is invalid.
+401 | 8 | `consumer_key_rejected` | The Consumer Key has incorrect length or does not exist.
+401 | 9 | `token_used` | An attempt of authorization of an already authorized token or an attempt to exchange a not temporary token for a permanent one.
+401 | 10 | `token_expired` | The temporary token has expired. At the moment, the mechanism of expiration of temporary tokens is not implemented and the current error is not used.
+401 | 11 | `token_revoke` | The token is revoked by the user who authorized it.
+401 | 12 | `token_rejected` | The token is not valid, or does not exist, or is not valid for using in the current type of request.
+401 | 13 | `verifier_invalid` |The confirmation string does not correspond to the token.
```
Comment (jeff-matthews) [inline guides/v2.1/get-started/authentication/oauth-errors.md]: Add the following inline attribute to tables per style guide

> {:style="table-layout:auto;"}

File: guides/v2.1/get-started/create-integration.md [hunk 5]
```diff
@@ -37,63 +33,54 @@ To develop a module, you must:
...
-               &lt;/sequence>
-          &lt;/module>
-        &lt;/config>
-   </pre>
-
-   Module `Magento_Integration` is added to "sequence" to be loaded first. It helps to avoid the issue, when a module with integration config loaded, that leads to a malfunction.
+    Attribute | Description
+    --- | ---
+    `name` | A string that uniquely identifies the module
+    `setup_version` | The version of Magento the component uses
```
Comment (jeff-matthews) [inline guides/v2.1/get-started/create-integration.md]: Add the following inline attribute to tables per style guide

> {:style="table-layout:auto;"}

File: guides/v2.1/get-started/create-integration.md [hunk 6]
```diff
@@ -215,55 +201,34 @@ In the following example, the test integration requires access to the following
...
-</table>
+```
+
+Element | Description
+--- | ---
+`integrations` | Contains one or more integration definitions.
+`integration name` | Defines an integration. The `name` must be specified.
+`email` | An email to associate with this integration.
+`endpoint_url` | Optional. The URL where OAuth credentials can be sent when using OAuth for token exchange. We strongly recommend using `https://`. See [OAuth-based authentication]({{ page.baseurl }}/get-started/authentication/gs-authentication-oauth.html) for details.
+`identity_link_url` | Optional. The URL that redirects the user to link their 3rd party account with the Magento integration.
```
Comment (jeff-matthews) [inline guides/v2.1/get-started/create-integration.md]: Add the following inline attribute to tables per style guide

> {:style="table-layout:auto;"}

File: guides/v2.1/get-started/gs-curl.md [hunk 7]
```diff
@@ -1,105 +1,35 @@
...
+The cURL examples in this guide use the following command-line options:
+
+Option | Description
+--- | ---
+`-d` | Sends the specified data in a POST request to the HTTP server. Use this option to send a JSON or XML request body to the server.
+`-H` | Specifies an extra HTTP header in the request. Precede each header with the `-H` option. You can specify any number of extra headers. For a list of common headers used in Magento web API requests, see [HTTP headers]({{ page.baseurl }}/get-started/gs-web-api-request.html#http-headers)
+`-i` | Includes the HTTP header in the output.
+`-s` | Specifies silent or quiet mode, which makes cURL mute. Progress and error messages are suppressed.
+`-T` | Transfers the specified local file to the remote URL.
+`-X` | Specifies the request method to use when communicating with the HTTP server. The specified request method is used instead of the default GET method.
```
Comment (jeff-matthews) [inline guides/v2.1/get-started/gs-curl.md]: Add the following inline attribute to tables per style guide

> {:style="table-layout:auto;"}

**Comment by keharper** [comment]
Added all requested table formatting.

**Pull request merged**